conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -432,17 +432,18 @@ TEST_F(
 
 class zetMetricQueryTest : public zetMetricGroupTest {
 protected:
-  zet_metric_query_pool_handle_t metricQueryPoolHandle;
-  zet_metric_query_handle_t metricQueryHandle;
+  zet_metric_query_pool_handle_t metricQueryPoolHandle{};
+  zet_metric_query_handle_t metricQueryHandle{};
 
   std::vector<std::string> groupNameList;
-  zet_metric_group_handle_t matchedGroupHandle;
+  zet_metric_group_handle_t matchedGroupHandle{};
   std::string groupName;
 
-  zetMetricQueryTest() {
-
+  void zetMetricQueryInitialize() {
     groupNameList = lzt::get_metric_group_name_list(
         device, ZET_METRIC_GROUP_SAMPLING_TYPE_FLAG_EVENT_BASED, false);
+    ASSERT_GT(groupNameList.size(), 0u)
+        << "metric group name list cannot be empty";
```

Comment:
instead of describing the condition, describe the error. e.g. "No query metric groups found". This simplifies understanding the error when looking at logs in CI

Reply:
same for all occurrences

Reply:
Resolved, Thank you.